runtime/v2/runner/src/main/java/com/walmartlabs/concord/runtime/v2/runner/script/DefaultScriptEvaluator.java

```diff
@@ -114,21 +114,34 @@ public String getLanguage(String languageOrExtension) {
         return null;
     }
 
+    private org.graalvm.polyglot.Context.Builder getGraalEngineContextBuilder(Map<String,Object> variables) {
+      HostAccess access = HostAccess.newBuilder(HostAccess.ALL)
+        .targetTypeMapping(Value.class, Object.class, Value::hasArrayElements, v -> new LinkedList<>(v.as(List.class))).build();
+      org.graalvm.polyglot.Context.Builder ctx = org.graalvm.polyglot.Context.newBuilder("js")
+        .allowHostAccess(access);
+      for (String key : variables.keySet()) {
+        switch (key) {
+          case "ecmascript-version":
+            ctx.option("js."+key, variables.get(key).toString());
```

Comment:
should we allow arbitrary values to get passed straight through here? Seems like it should come from a pre-existing list of options or at least go through some validation. It _should_ bonk when graalvm doesn't accept the value and gives a better-than-nothing error
```text
Error: For input string: "2022z"
```

Reply:
yep, absolutely we should do that.

see: 9b6bc101e